packages/manager/modules/vps/src/dashboard/terminate/translations/Messages_fr_FR.json

```
@@ -4,5 +4,6 @@
   "vps_terminate_cancel": "Annuler",
   "vps_terminate_confirm": "Confirmer",
   "vps_terminate_success": "La suppression du service a été prise en compte, vous recevrez un e-mail de confirmation pour valider la suppression.",
-  "vps_terminate_error": "Une erreur est survenue lors de la suppression du service."
+  "vps_terminate_error": "Une erreur est survenue lors de la suppression du service.",
+  "vps_terminate_unavailable": "Vous avez souscrit cet abonnement avec un engagement de {{ monthCount }} mois. Si vous avez la moindre question, n’hésitez pas à contacter notre service client par chat ou en envoyant un Ticket d'assistance. Merci de votre compréhension"
```

```suggestion
  "vps_terminate_unavailable": "Vous avez souscrit cet abonnement avec un engagement de {{ monthCount }} mois. Si vous avez la moindre question, n’hésitez pas à contacter notre service client par chat ou en envoyant un ticket d'assistance. Merci de votre compréhension"
```
capital letter looks unecessary here